activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+*   Add `after_or_equal?` and `before_or_equal?` methods to `Date`, `DateTime`, `Time`, and `TimeWithZone`.
+
+    These methods provide a more readable alternative to using `>=` and `<=` when comparing dates and times.
```

Comment:
Is `time1.after_or_equal?(time2)` is actually more readable than `time1 >= time2`? 😕